components/dropdown/Dropdown.js

```diff
@@ -157,6 +167,7 @@ const factory = (Input) => {
             {...others}
             className={theme.value}
             onClick={this.handleClick}
+            required={this.props.required}
             readOnly
```

Comment:
I wanted to validate a form with html5 constraint validation while setting the required property. 

Applying the `readonly` treats the input as valid, making the required attribute pointless.

I  want to get rid of the readonly attribute

context: https://stackoverflow.com/questions/12777751/html-required-readonly-input-in-form